brambling/views/orders.py

```diff
@@ -823,6 +823,28 @@ def get_context_data(self, **kwargs):
                 'dwolla_user_id': account.user_id
             })
         context.update(self.summary_data)
+
+        # TODO: Improve this overall assembly. It's possible the data model
+        # will need to change to allow this. Invite doesn't have a reference
+        # back to its content though, just an IntegerField content_id.
+        bought_items = self.order.bought_items.filter(status=BoughtItem.BOUGHT)
+        invites = TransferInvite.get_invites(bought_items)
+        context['pending_transfers'] = [
+            dict(
+                invite=invite,
+                bought_item=bought_items.get(pk=invite.content_id),
+            )
+            for invite in invites
+        ]
+
+        # This is pretty awful and hopefully only used as a stopgap measure
+        # before Invites get refactored. Then again, this comment is one of
+        # those kinds that stick around for years.
+        context['transferred_items'] = BoughtItem.objects.filter(
+            transactions__related_transaction__transaction_type=Transaction.TRANSFER,
+            transactions__related_transaction__order=self.order,
+        )
```

Comment:
I've gone ahead and done this, but I think I disagree because the number of items is small (and thus the number of queries) and `BoughtItem#get_transfer` was more readable, IMO.